README.md

```diff
@@ -13,13 +13,8 @@ This process is clunky right now, so my apologies.
 
 1. Clone the repo: `git clone https://github.com/GUI/covid-vaccine-spotter.git`
 2. Install dependencies (inside the repo): `yarn install`
-3. Download data from the [API](https://www.vaccinespotter.org/api/) for the website to run. Note these example steps just download a few files to get things running, but it's not an entire dataset. You can download other states data, but I know this is currently a bit cumbersome.
-   - `mkdir -p site/api/v0/stores/CO`
-   - `curl -o site/api/v0/states.json https://www.vaccinespotter.org/api/v0/states.json`
-   - `curl -o site/api/v0/stores/CO/cvs.json https://www.vaccinespotter.org/api/v0/stores/CO/cvs.json`
-   - `curl -o site/api/v0/stores/CO/walgreens.json https://www.vaccinespotter.org/api/v0/stores/CO/walgreens.json`
-   - `curl -o site/api/v0/stores/CO/walmart.json https://www.vaccinespotter.org/api/v0/stores/CO/walmart.json`
-4. To run the development server for the website: `yarn run eleventy --serve`. The development site should then be available at http://localhost:8080/.
+3. Fetch data from the [API](https://www.vaccinespotter.org/api/) for the website to run: `yarn run setup`
+4. To run the development server for the website: `yarn run start`. The development site should then be available at http://localhost:8080/.
```

Comment:
I followed these instructions but got the message that I don't have a dist/ folder. So I had to run `yarn generate` to generate the dist/ folder. Useful to add into the README

Reply:
Thanks @rotywonka, I will update the README with this info